### ISSUE TITLE
The "ZAP" example for CR 5 contradicts itself

### ISSUE THREAD

**Issue by mitchellevan** [w3c/wcag#632]
In Understanding Conformance, the section "Understanding Requirement 5" has an example with this sentence: "Although ZAP is accessibility-supported, the information that is presented in ZAP is also presented on the page in HTML, so ZAP is not relied upon." A word is missing, which causes the sentence to contradict itself.

The word "not" should be added. The sentence should be: "Although ZAP is not accessibility-supported, the information that is presented in ZAP is also presented on the page in HTML, so ZAP is not relied upon."

The problem is in this page:
https://www.w3.org/WAI/WCAG21/Understanding/conformance

I also reported this issue in WCAG 2.0 Understanding Conformance.

**Comment by alastc** [comment]
Looks like this needs a 2.1 overhaul in general, and I agree the 'not' is missing. Assigned to myself.

**Comment by awkawk** [comment]
It is also correct as it is, although it doesn't make the point as well.

**Comment by awkawk** [comment]
Proposed WG Response: Thank you for the comment. This error is addressed in Pull Request #654

**Comment by awkawk** [comment]
Agreed on March 26 call: https://www.w3.org/2019/03/26-ag-minutes.html#item09
